console_connect_four.py

```diff
@@ -2,11 +2,12 @@
 from core.board import Board
 from core.player import Player
 from core.strategy_loader import StrategyLoader
+from tournaments.round_robin_tournament import RoundRobinTournament
 from views.zelle_graphics_game_view import ZelleGraphicsGameView
+from views.console_game_view import ConsoleGameView
 
 strategy_loader = StrategyLoader(strategies_dir="strategies")
-strategy_loader.find_and_load()
-player1 = Player(name="Bot", strategy=strategy_loader[0])
-player2 = Player(name="Player", strategy=strategy_loader[1])
-game = ConnectFourGame(Board(width=7, height=6))
-game.play(player1, player2, ZelleGraphicsGameView())
+tournament = RoundRobinTournament(strategy_loader, ConsoleGameView())
+results = tournament.run()
+print(results)
```

Comment:
It would be nice to either equip a game view to be able to print results at either a game or tournament level, or introduce a tournament view that works alongside the game view.